quiz.cpp

```diff
@@ -34,11 +34,11 @@ QString Quiz::getQuizName()
     return m_quizName;
 }
 
-void Quiz::setCollectionOfQuestions(std::vector<Question>& collectionOfQuestions)
+void Quiz::setCollectionOfQuestions(std::vector<Question> collectionOfQuestions)
 {
     m_collectionOfQuestions = collectionOfQuestions;
 }
-std::vector<Question> Quiz::getCollectionOfQuestions()
+std::vector<Question>& Quiz::getCollectionOfQuestions()
```

Comment:
Kod getter metoda fali ``const`` nakon zagrade: https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/const.md

Ona je garant da se ništa u getter metodi neće menjati, evo primera koji se ne kompajlira zato što je getter metoda obeležena sa ``const``:

```
class Foo
{
public:
        int getBar() const {
                bar = 3;
                return bar;
        }
private:
        int bar;
};

int main()
{
        return 0;
}
```